packages/catalog-search/src/SearchSuggestionItem.jsx

```diff
@@ -45,7 +45,7 @@ SearchSuggestionItem.propTypes = {
     key: PropTypes.string,
     title: PropTypes.string,
     program_type: PropTypes.string,
-    authoring_organizations: PropTypes.array,
+    authoring_organizations: PropTypes.shape([]),
```

Comment:
`PropTypes.shape` is usually reserved for an object prop, not an array. Should this be `PropTypes.arrayOf(PropTypes.shape())` instead?

Reply:
Related, is this `hit.authoring_organizations` actually used in this component? If not, is it necessary to include in the prop types definition?

Reply:
`authoring_organizations` is not used in this component. Removed it from prop types definition